README.md

```diff
@@ -1,5 +1,6 @@
 # Chainguard Images
 
+
```

Comment:
you can drop this

ran locally and and there is not difference, so we can ignore this check for now

Reply:
see slack: https://chainguard-dev.slack.com/archives/C0322APD4LU/p1670235759571799

Reply:
OK, I'll try to roll back...